### PR TITLE
add react-disqus-thread v0.3.1

### LINKED ISSUE #7309: [Request] Add react-disqus-thread
**Library name:** react-disqus-thread
**Git repository url:**https://github.com/mzabriskie/react-disqus-thread
**npm package url(optional):** https://www.npmjs.com/package/react-disqus-thread
**License(s):** MIT
**Official homepage:**http://react-components.com/component/react-disqus-thread
**Wanna say something? Leave message here:** Really appreciate the work you do!

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by pvnr0082t**
PR for #7309 
@PeterDaveHello Please help me review this pull request, thank you.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/mzabriskie/react-disqus-thread
 * Official website (optional, not the repository): http://react-components.com/component/react-disqus-thread
 * NPM package url (optional): https://www.npmjs.com/package/react-disqus-thread
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 39
   - Count of watchers: 4
   - Count of forks: 18
 * NPM download stats (optional):
   - Downloads in the last day: 15
   - Downloads in the last week: 217
   - Downloads in the last month: 1147

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

with git autoupdate config, close #7309, cc @mzabriskie

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
It needs manually build, so you should try:
https://github.com/mzabriskie/react-disqus-thread/tree/v0.3.1/dist

**Comment by pvnr0082t** [comment]
@PeterDaveHello I should use git autoupdate?

**Comment by PeterDaveHello** [comment]
@pvnr0082t Yes!

**Comment by pvnr0082t** [comment]
@PeterDaveHello I have already fixed with git autoupdate, thank you.

File: ajax/libs/react-disqus-thread/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "author": {
+    "name": "Matt Zabriskie"
+  },
```
Comment (PeterDaveHello) [inline ajax/libs/react-disqus-thread/package.json]: Use `"author": "Matt Zabriskie"` is easier since there is only "name" in it, please try it next time.

Comment (pvnr0082t) [inline ajax/libs/react-disqus-thread/package.json]: OK, I got it.

**Pull request merged**